test/extensions/quic_listeners/quiche/platform/quic_platform_test.cc

```diff
@@ -92,13 +96,13 @@ TEST(QuicPlatformTest, QuicHostnameUtils) {
 }
 
 TEST(QuicPlatformTest, QuicUnorderedMap) {
-  quic::QuicUnorderedMap<quic::QuicString, int> umap;
+  quic::QuicUnorderedMap<std::string, int> umap;
```

Comment:
Remove all quic:: from this file? (but let's resolve my comment on test/test_common/logging.h before doing so)

Reply:
done.